### ISSUE TITLE
pkgconfig files in /usr/share complain

### ISSUE THREAD

**Issue by jengelh** [rpm-software-management/rpmlint#121]
In openSUSE:Factory, rpmlint-mini-1.0 says:
```
fish.x86_64: E: devel-file-in-non-devel-package (Badness: 50) /usr/share/pkgconfig/fish.pc
A file that is needed only e.g. when developing or building software is
included in a non-devel package. These files should go in devel packages.
```
was observed. However, share/pkgconfig files do not count as development.

**Comment by Conan-Kudo** [comment]
@jengelh Why are they not considered as development files? `pc` files are _exclusively_ used by software build systems, such as Autotools, Meson, and CMake. I know of no tooling to leverage `pc` files in a non software development case.

**Comment by scarabeusiv** [comment]
@jengelh could you find some documentation that says that the /usr/share/pkgconfig is not considered development content?

If we look on my machine:

```
scarabeus@bugaboo: ~ $ LANG=C ls -1 /usr/share/pkgconfig
adwaita-icon-theme.pc
applewmproto.pc
bash-completion.pc
bigreqsproto.pc
compositeproto.pc
damageproto.pc
dmxproto.pc
dracut.pc
dri2proto.pc
dri3proto.pc
fixesproto.pc
fontsproto.pc
glproto.pc
inputproto.pc
kbproto.pc
mobile-broadband-provider-info.pc
personality.d/
poppler-data.pc
presentproto.pc
randrproto.pc
recordproto.pc
renderproto.pc
resourceproto.pc
scrnsaverproto.pc
shared-mime-info.pc
systemd.pc
udev.pc
videoproto.pc
windowswmproto.pc
xcmiscproto.pc
xextproto.pc
xf86bigfontproto.pc
xf86dgaproto.pc
xf86driproto.pc
xf86vidmodeproto.pc
xineramaproto.pc
xkeyboard-config.pc
xproto.pc
yelp-xsl.pc
```

And checking ie xproto:

```
scarabeus@bugaboo: ~ $ rpm -qf /usr/share/pkgconfig/xproto.pc
xorgproto-devel-2019.1-1.1.x86_64
```

So I don't think the statement is wrong.

**Comment by Conan-Kudo** [comment]
`/usr/share/pkgconfig` is for architecture independent pkgconfig files. It's supported by all pkgconfig implementations. Thus, I do not consider this an issue.
